### ISSUE TITLE
No multiple window star/trajectory/background rendering size consideration.

### ISSUE THREAD

**Issue by eriksunden** [OpenSpace/OpenSpace#458]
The stars/trajectory FBO size is set to that one of the last window. In a multiple window rendering scenario on one client all windows need to have the same FBO size, for it to look correct.
For instance, create a dual window SGCT config, with a lower size window of the second window, does result in the output of the attached screenshot (red corners indicate top corners of this rendering).
![openspace_multiwindow_differentsize_bug](https://user-images.githubusercontent.com/8737339/34758570-5a78ddf2-f5d7-11e7-8ca8-64a9793c7915.png)

**Comment by GPayne** [comment]
The steps to reproduce are a little hard to follow.
Is there an expected fix for this, or is it unknown how to address this problem?
